sdk/storage/azblob/blob/models.go

```diff
@@ -146,11 +155,11 @@ func (o *DownloadToWriterAtOptions) getDownloadBlobOptions(offSet, count int64,
 	}
 }
 
-// DownloadToBufferOptions identifies options used by the DownloadToBuffer and DownloadToFile functions.
-type DownloadToBufferOptions = DownloadToWriterAtOptions
+// DownloadBufferOptions identifies options used by the DownloadBuffer and DownloadFile functions.
+type DownloadBufferOptions = DownloadWriterAtOptions
```

Comment:
This type, and `DownloadFileOptions`, should be discrete types so that they can independently evolve over time.  They can anonymously embed the `DownloadWriterAtOptions`.

Reply:
Hmm, I think it makes sense. Alright.